### PR TITLE
mention the smørrebrød at riccos

### PR DESCRIPTION
**Pull request by mafintosh**

### CODE REVIEW

**Comment by mikker** [comment]
Don't forget the egg one! Homemade mayo.

**Comment by Munter** [comment]
Didn't they just split up and rename the smørrebrød serving café's something different?

File: README.md [hunk 1]
```diff
@@ -225,6 +225,8 @@ Wifi | Power | Address | Open Hours
 
 #### Riccos Kaffebar (Sluseholmen)
 
+> Make sure to get the rhy bread sandwiches. Their advocadomad+kartoffelmad are sublime
```
Comment (RichardLitt) [inline README.md]: Is rye bread spelled rhy in Denmark?

Comment (mafintosh) [inline README.md]: rugbrød (good luck pronouncing that btw) - i'll update the pr

**Comment by mafintosh** [comment]
@Munter not at my local riccos

**Comment by RichardLitt** [comment]
Thanks!

**Comment by diasdavid** [comment]
sweet, thank you @mafintosh  :D

**Pull request merged**